README.md

```diff
@@ -46,28 +141,28 @@ jobs:
           file: doc/api-documentation.yml
 ```
 
-Important: [actions/checkout](https://github.com/actions/checkout) has to be called **before this action**.
-
+_Important: make sure to change your main destination branch name (`main` in the example above) and replace `<BUMP_DOC_ID>` with your Bump documentation slug or id._
 
 ## Inputs
 
-* `doc` (required): Documentation id or slug. Can be found in the documentation settings on https://bump.sh
+* `doc` (required): Documentation slug or id. Can be found in the documentation settings on https://bump.sh
```

Comment:
Maybe we should favor direct access to front office (I'm quite sure that's what we do in other place but I can't find where)
```suggestion
* `doc` (required): Documentation slug or id. Can be found in the documentation settings on https://bump.sh/docs
```

Reply:
clearly!